VulkanHppGenerator.cpp

```diff
@@ -7661,6 +7662,12 @@ std::string VulkanHppGenerator::generateHandle( std::pair<std::string, HandleDat
 
 ${CppType}
 
+  template <>
+  struct CppType<Vk${className}, VK_NULL_HANDLE>
```

Comment:
On 32-bit builds non-dispatchable handles boil down to a 64bit int type.
So these end up as conflicting templates, which causes compilation failure.